### PR TITLE
created the db_build.sql

### PR DESCRIPTION
**Pull request by FarahZaqout**

### CODE REVIEW

File: src/database/db_build.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
+BEGIN;
+
+DROP TABLE IF EXISTS books;
```
[the remark this answers is not in the document]
Comment (FarahZaqout) [reply src/database/db_build.sql]: Done

File: src/database/db_build.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+DROP TABLE IF EXISTS reservation;
+
+CREATE TABLE books(
+id SERIAL PRIMARY KEY,
+name VARCHAR(50),
```
[the remark this answers is not in the document]
Comment (FarahZaqout) [reply src/database/db_build.sql]: Done

File: src/database/db_build.sql [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+author VARCHAR(50),
+);
+
+CREATE TABLE users(
+id INTEGER,
```
[the remark this answers is not in the document]
Comment (FarahZaqout) [reply src/database/db_build.sql]: Done

File: src/database/db_build.sql [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+name VARCHAR(50),
+phone INTEGER,
+email VARCHAR(50),
```
[the remark this answers is not in the document]
Comment (FarahZaqout) [reply src/database/db_build.sql]: Done

File: src/database/db_build.sql [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+user_name VARCHAR(50),
```
[the remark this answers is not in the document]
Comment (FarahZaqout) [reply src/database/db_build.sql]: Done

File: src/database/db_build.sql [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+password VARCHAR(50)
+);
+
+CREATE TABLE reservation(
+book_id INTEGER FOREIGN KEY(books),
```
[the remark this answers is not in the document]
Comment (FarahZaqout) [reply src/database/db_build.sql]: correct way turned out to be: REFERENCES books(id)

File: src/database/db_build.sql [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+user_id INTEGER FOREIGN KEY(users),
```
[the remark this answers is not in the document]
Comment (FarahZaqout) [reply src/database/db_build.sql]: ditto

**Comment by IsraaMoqbel** [comment]
great job!

**Pull request merged**